src/plugins/github/fetchReposForOrg.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import {type RepoId, makeRepoId} from "../../core/repoId";
+import dedent from "../../util/dedent";
+
+const GITHUB_GRAPHQL_SERVER = "https://api.github.com/graphql";
+async function ghquery(query, token): any {
```

Comment:
Why not use `postQuery` from `fetchGithubRepo.js`? That function already
exists and also has better error checking (rate limits, internal
execution errors, …) and the correct return type (`Promise<any>`).

(Also, `ghquery` is a convenient name for a shell alias, but a JS
function deserves a clearer name, like `postQuery`. :-) )